packages/ui/src/forum/components/Thread/ThreadNewPost.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react'
+
+import { ButtonPrimary, ButtonsGroup } from '@/common/components/buttons'
+import { CKEditor } from '@/common/components/CKEditor'
+import { InputComponent } from '@/common/components/forms'
+import { TextBig } from '@/common/components/typography'
+import { useModal } from '@/common/hooks/useModal'
+import { CreatePostModalCall } from '@/forum/modals/PostActionModal/CreatePostModal'
+import { ForumThread } from '@/forum/types'
+import { useMyMemberships } from '@/memberships/hooks/useMyMemberships'
+
+import { EditorWrap } from '../PostList/PostEditor'
+
+interface NewPostProps {
+  thread: ForumThread
+}
+
+export const NewThreadPost = ({ thread }: NewPostProps) => {
+  const [postText, setText] = useState('')
+  const { active } = useMyMemberships()
+  const { showModal } = useModal()
+
+  if (!active) {
+    return <TextBig>Pick an active membership to post in this thread</TextBig>
+  }
+
+  return (
+    <EditorWrap>
+      <InputComponent inputSize="auto">
+        <EditorMemo setNewText={setText} />
+      </InputComponent>
+      <ButtonsGroup>
```

Comment:
This or the input should be wrapped in something (RowGapBlock?) in order to give some space to the button.